tests/integration/targets/sts_assume_role/tasks/main.yml

```diff
@@ -51,33 +51,6 @@
            - 'result.failed'
            - "'missing required arguments:' in result.msg"
 
-    # ============================================================
-    - name: test with empty parameters
-      sts_assume_role:
-        role_arn:
```

Comment:
Ansible now rejects None if it's supposed to be a string (YAML returns a Nil for blank entries which aren't explicitly passed as an empty string).